backend/src/app.py

```diff
@@ -1,13 +1,13 @@
-from fastapi import FastAPI
-from .api import chat
+from . import create_app
 from mangum import Mangum
-import uvicorn
 
-app = FastAPI()
-app.include_router(chat.router)
+app = create_app()
 
-@app.get("/")
-def get_root():
-    return {"message": "FastAPI running in a Lambda function"}
+# Generate Swagger schema
+# import json
+# openapi_schema = app.openapi()
+# openapi_schema_json = json.dumps(openapi_schema, indent=2)
+# with open('openapi_schema.json', 'w') as file:
+#     file.write(openapi_schema_json)

```

Comment:
Why comment out? I like the idea of swagger. If this is not finished, I would remove the comments from here and place it in another branch.

Reply:
Just wanted to prevent generating the swagger schema every time the backend starts locally, I have created the schema here [https://github.com/access2justice/law-bot/blob/aws/add_azure_clients/backend/openapi_schema.json](url)

Reply:
Can we set the activation of swagger with an environment variable instead? Using these comments feels somewhat hacky.